themes/base.theme.bash

```diff
@@ -408,7 +408,13 @@ function safe_append_prompt_command {
     if [[ -n $1 ]] ; then
         case $PROMPT_COMMAND in
             *$1*) ;;
-            *) PROMPT_COMMAND="$1;$PROMPT_COMMAND";;
+            *)
```

Comment:
Could you change this to the following, which should match the empty string?

```bash
"") PROMPT_COMMAND="$1";;
*) PROMPT_COMMAND="$1;$PROMPT_COMMAND";;
```

The `"")` should match an empty value - this way it's cleaner and does not require a nested `if`.